### ISSUE TITLE
Can not read the property `crypto` of undefined on Web Worker scope

### ISSUE THREAD

**Issue by tresdin** [fpirsch/twin-bcrypt#4]
Please change this line
```
        factory(root.TwinBcrypt = {}, this.crypto || this.msCrypto);
```
to
```
        factory(root.TwinBcrypt = {}, self.crypto || self.msCrypto);
```
Otherwise, it fails on Web Worker scope.

**Comment by fpirsch** [comment]
Can you be more specific (which browser are you using, what kind of failure or error message) ?
`self` doesn't exist in a Node.js context, but `this` refers to the global context in the browser (both main window and workers) and in Node. And definitely works in web workers, as you can see [here](http://aspirine.org/htpasswd.html)

**Comment by tresdin** [comment]
My bad. It's actually fails on strict mode. You can try your code on strict mode and see the result. I've tested the code with `self` and it works fine on node.

For more information, please see this article.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this#Function_context

**Comment by fpirsch** [comment]
Strict mode. Of course. Thanks !
Fixed by 572e98a09c
